FirebaseABTesting/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v4.2.0
```

Comment:
Will be 7.0.0

Reply:
Resolved in 4cce114.

I changed the base to `firebase7-main` but it ended up pulling in a bunch of cruft into the commits. Did I mess something up?

Reply:
Yeah, probably the best thing is to rebase your commits onto the firebase7-main branch